<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BA_MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="GraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="8" column="0">
       <widget class="QToolButton" name="toolButton_zoom_plus">
        <property name="text">
         <string>Zoom+</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QSpinBox" name="spinBox_nd_s">
        <property name="maximum">
         <number>50000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="nd_pushButton">
          <property name="text">
           <string>nd(s,t)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fd_pushButton">
          <property name="text">
           <string>fd(s,t)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rout_pushButton">
          <property name="text">
           <string>rout(s,t)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="1">
       <widget class="QToolButton" name="toolButton_zoom_minus">
        <property name="text">
         <string>Zoom-</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QSpinBox" name="vertNoSpinBox">
        <property name="maximum">
         <number>2000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="selVertButton">
        <property name="text">
         <string>select highlight vertex</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QSpinBox" name="size_spinBox">
        <property name="maximum">
         <number>2000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinBox_nd_t">
        <property name="maximum">
         <number>50000</number>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="importRoutePushButton">
          <property name="text">
           <string>import Route</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="importButton">
          <property name="text">
           <string>import Hist</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>605</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHistogramm">
    <property name="title">
     <string>Histogramm</string>
    </property>
    <addaction name="actionreset"/>
    <addaction name="actiongenerate_new_histogramm"/>
    <addaction name="actionreset_rout"/>
    <addaction name="actionoptimal_route"/>
   </widget>
   <widget class="QMenu" name="menuStats">
    <property name="title">
     <string>Stats</string>
    </property>
    <addaction name="actiongen_Data"/>
   </widget>
   <addaction name="menuHistogramm"/>
   <addaction name="menuStats"/>
  </widget>
  <action name="actionreset">
   <property name="text">
    <string>reset_highlighted_vertex</string>
   </property>
  </action>
  <action name="actiongenerate_new_histogramm">
   <property name="text">
    <string>generate_new_histogramm</string>
   </property>
  </action>
  <action name="actionreset_rout">
   <property name="text">
    <string>reset_rout</string>
   </property>
  </action>
  <action name="actionDaten_sammeln">
   <property name="text">
    <string>Daten sammeln</string>
   </property>
  </action>
  <action name="actiongen_Data">
   <property name="text">
    <string>gen_Data</string>
   </property>
  </action>
  <action name="actionoptimal_route">
   <property name="text">
    <string>optimal_route</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>graphicsview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>vertNoSpinBox</tabstop>
  <tabstop>spinBox_nd_s</tabstop>
  <tabstop>spinBox_nd_t</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
